docs/Antora/modules/background_info/pages/streaming.adoc

```diff
@@ -1,10 +1,67 @@
+which may be connected to xref:glossary:glossary.adoc#input_port[Input Ports].
+
+The generalized Streaming mechanism serves to decouple data streaming from the Device structure publishing.
+
+== Streaming instantiation
+
+A specific Streaming protocol implementation is associated with an openDAQ(TM) xref:background_info:modules.adoc[Module].
+The SDK handles the routing and delegation of Streaming object instantiation to the appropriate module
+through using the connection string. The connection string always starts with the protocol type ID as its
+prefix, followed by a set of parameters unique to the protocol type. Some parameters, such as the IP address,
+might be obligatory, while others, like the port number, are optional and adopt default values if not
+explicitly specified.
+
+The openDAQ(TM) supports two types of xref:glossary:glossary.adoc#device[Device]s: those with a complete tree structure and those (also called
+a pseudo-devices) with a flat list of xref:glossary:glossary.adoc#signal[Signal]s. Regardless of the Device
+type, upon connection, it should be able to immediately start the data streaming. To accomplish this,
+the client Module responsible for creating the Device object also instantiates a Streaming object.
+It is achieved in two ways:
+
+* for pseudo-devices with a flat list of signals the module creates a Streaming object on its own using
+the connection parameters specified in the device connection string.
+* for devices supporting a complete tree structure, the module can delegate the creation of the Streaming
+object to another module using the information about supported Streaming protocols published by the Device.
+
+The above also implies that, when connecting to a Device, all its streaming capabilities are discovered,
+including those of its sub-devices, making it possible to establish connections with all of them,
+enabling flexibility in choosing which to use. However, there also exists a configuration mechanism to
+control and limit the automatic establishment of streaming connections during device connection.
+The configuration mechanism enables the filtering of Streaming capabilities available per device based on
+protocol type. Additionally, it provides control over the number of hops the streamed data must make
+between nested devices.
+
+* *How-to:* xref:howto_guides:howto_configure_streaming.adoc#streaming_config_per_device[Configure Streaming per Device]
+
+[#mirrored_signals]
+== Mirrored signals
+
+A Mirrored Signal serves as a mirrored representation of the xref:glossary:glossary.adoc#signal[Signal]
+originating from a remote xref:glossary:glossary.adoc#device[Device]. While it aids users in examining
+and configuring the Signals of a remote Device, each Mirrored Signal can be individually linked to a
+corresponding Signal on the Device through a selected Streaming protocol, establishing a connection that
+allows the real-time transfer of data from the openDAQ(TM) Device to the openDAQ(TM) Client.
```

Comment:
This is a bit hard to read/understand. 

I'd clarify that mirrored signals allow for inspecting the configuration and setup of the remote device (structure), and also act as the medium for receiving device data on a client. Each mirrored signal has a list of streaming sources based on established streaming connections. Among those, users can choose a signal to act as the active streaming source - only that one will send data through the signal.

(note that my comment is not cleaned up in terms of wording/grammar)